src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/PostgreSqlDataSourceItemFixture.cs

```diff
@@ -0,0 +1,40 @@
+using Reveal.Sdk.Dom.Core.Extensions;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class PostgreSqlDataSourceItemFixture
+    {
+        [Fact]
+        public void Constructor_SetsTitleAndDataSource_WhenConstructed()
+        {
+            // Arrange
+            var title = "Test Item";
+            var dataSource = new PostgreSQLDataSource();
+
+            // Act
+            var item = new PostgreSqlDataSourceItem(title, dataSource);
+
+            // Assert
+            Assert.Equal(title, item.Title);
+            Assert.Equal(dataSource, item.DataSource);
```

Comment:
Should use [Theory]
Need more Asserts

DataSourceId = dataSource.id
title = DataSouce.Title if DataSource.Title == null